frontend/src/components/VideoCardPage/styles.css

```diff
@@ -0,0 +1,12 @@
+.root {
+    overflow: hidden;
+}
+
+.card_container {
+    margin: auto !important;
```

Comment:
You know that !important is a bad practice. No way around it?

Reply:
Not that I found